src/index.ts

```diff
@@ -19,25 +19,27 @@ async function runMain() {
     const buildParameters = await BuildParameters.create();
     const baseImage = new ImageTag(buildParameters);
 
+    let exitCode = 0;
```

Comment:
Defaulting exit code to 0 as a success seems right, but is there any chance that one of the async calls wouldn't finish and having a value in `exitCode` would then be misleading? I'm asking but I really don't know what would be better between 0 and null or undefined.